### PR TITLE
Update for latest api

### PR DESCRIPTION
**Pull request by justmoon**
Removes universal mode, updates for latest Ripple API and ILP plugin interface.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,22 +1,17 @@
...
-
-    TRADER_BACKEND='fixerio-plus-xrp'
-
+**TODO** Add more explanation
+
+``` json
+{
+  "type": "ripple",
+  "server": "wss://s.altnet.rippletest.net:51233",
+  "address": "r33L6z6LMD8Lk39iEQhyXeSWqNN7pFVaM6",
```
Comment (bensharaf) [inline README.md]: prefix?

Comment (justmoon) [inline README.md]: Hardcoded to be `ripple.`

File: index.js [hunk 2]
```diff
@@ -1,101 +1,136 @@
...
-  }.bind(this))
-}
+class RipplePlugin extends EventEmitter {
+  constructor (options) {
+    super()
 
-RippledLedger.TYPE = 'https://ripple.com/ilp/v1'
-RippledLedger.validateTransfer = require('./lib/validate')
+    this.server = options.server
+    this.address = options.address
```
Comment (bensharaf) [inline index.js]: should address include prefix?

Comment (justmoon) [inline index.js]: Options shouldn't include the prefix. But when calling `getAccount` it'll be included.

**Comment by bensharaf** [comment]
LGTM

File: index.js [hunk 3]
```diff
@@ -1,101 +1,136 @@
...
+      // we can do it by sending a subscribe request manually.
+      this.api.connection.request({
+        command: 'subscribe',
+        accounts: [this.address]
+      }).catch((err) => {
+        console.warn('ilp-plugin-ripple: unable to subscribe to transactions: ' +
+          (err && err.toString()))
+      })
+
+      this.emit('connect')
```
Comment (emschwartz) [inline index.js]: Shouldn't this only be emitted if it subscribes successfully?

Comment (emschwartz) [inline index.js]: Shouldn't this only be emitted if it subscribes successfully?

File: index.js [hunk 4]
```diff
@@ -1,101 +1,136 @@
...
+  }
 
-RippledLedger.prototype._rejectTransfer = function * (transfer) {
-  utils.setTransferState(transfer, 'rejected')
-  yield this.client.suspendedPaymentCancel(transfer)
-}
+  getInfo () {
+    return Promise.resolve({
+      scale: 2,
+      precision: 4
```
Comment (emschwartz) [inline index.js]: Isn't it higher than this for XRP?

**Pull request merged**